src/device.hpp

```diff
@@ -283,8 +283,27 @@ struct cvk_device : public _cl_device_id,
     }
 
     cl_uint sub_group_size() const {
+        if (supports_controled_subgroups()) {
+            cl_uint force_subgroup_size = config.force_subgroup_size();
```

Comment:
Since we're repeating the use of `config.` I'm not sure this variable adds much.